variables.tf

```diff
@@ -761,3 +761,12 @@ variable "kdump_crash_kernel_memory" {
   description = "The crashkernel memory reservation for kdump occurs during the system boot"
   default     = "2G-4G:384M,4G-16G:512M,16G-64G:1G,64G-128G:2G,128G-:4G"
 }
+
+###########################################
+# HAProxy API Server HealthCheck variable
+###########################################
+variable "haproxy_apiserver_healthcheck" {
+  type        = bool
+  description = "Flag to enable the haproxy_apiserver_healthcheck which will enable features for haproxy"
+  default     = true
```

Comment:
Are you sure we need this by default?

Reply:
Yes, it's the expected method used by OpenShift and included in many tests.